### PR TITLE
More polished tests

### PR DESCRIPTION
**Pull request by asinghvi17**

### CODE REVIEW

**Comment by asinghvi17** [comment]
Well, Gitlab's going to take a while...

File: examples/statsmakie.jl [hunk 1]
```diff
@@ -183,7 +183,7 @@
     end
 
     @cell "Errorbar" [errorbar] begin
-        using StatsMakie, Makie
```
Comment (SimonDanisch) [inline examples/statsmakie.jl]: That's an important change... Is this ready to merge?

Comment (asinghvi17) [reply examples/statsmakie.jl]: Yep!

Comment (asinghvi17) [reply examples/statsmakie.jl]: You'll probably need to merge, I'm on my phone and Guthub doesn't let you merge a failing PR from a phone

**Comment by asinghvi17** [comment]
Thanks!

**Pull request merged**